src/SUMMARY.md

```diff
@@ -1,14 +0,0 @@
-# Table of contents
```

Comment:
Why did we delete this file?

Reply:
As per your request to remove table of contents